.github/CODEOWNERS

```diff
@@ -2,7 +2,6 @@
 # We prefer to explicitly specify owners for all subdirectories.
 *                                @warner @Chris-Hibbert @FUDCo @michaelfig
 
-/*                               @kriskowal @michaelfig @erights
```

Comment:
My intent in proposing this line was to catch only the files in the root of the repository.  If it does more than that, I'm happy to remove it, otherwise, let's leave it in.

Reply:
Aha, yes. Also corroborated by https://github.com/hmarr/codeowners  

https://github.com/Agoric/agoric-sdk/pull/4966 and uninstalled the buggy extension =)